examples/src/main/java/io/dapr/examples/workflows/DemoWorkflowClient.java

```diff
@@ -45,22 +36,16 @@ public class DemoWorkflowClient {
    * @throws InterruptedException If program has been interrupted.
    */
   public static void main(String[] args) throws InterruptedException, TimeoutException {
-    DurableTaskClient client = new DurableTaskGrpcClientBuilder().build();
-    try (client) {
-      String name = DemoWorkflow.class.getCanonicalName();
-      String instanceId = client.scheduleNewOrchestrationInstance(name);
-      System.out.printf("Started new orchestration instance: %s%n", instanceId);
-
-      TimeUnit.SECONDS.sleep(5);
+    DurableTaskClient innerClient = new DurableTaskGrpcClientBuilder().build();
+    DaprWorkflowClient client = new DaprWorkflowClient(innerClient);
 
-      client.raiseEvent(instanceId, "myEvent");
-
-      OrchestrationMetadata completedInstance = client.waitForInstanceCompletion(
-          instanceId,
-          Duration.ofSeconds(5),
-          false);
+    try (client) {
+      String workflowName = "ExampleWorkflow";
+      String instanceId = client.scheduleNewWorkflow(workflowName);
 
-      System.out.printf("Orchestration completed: %s%n", completedInstance);
+      System.out.printf("Started new workflow instance: %s%n", instanceId);
+      // Workflow starts, completes
+      System.out.printf("Completed workflow: %s%n", instanceId);

```

Comment:
I would remove this since we have no methods to indicate that the workflow has completed.

It may be worth outlining what a good example would look like and leaving it commented out until the client has implemented those functions.

```java
// TimeUnit.SECOND.sleep(5);
// client.raiseEvent("myEvent", instanceId);
// 
// String instanceToTerminateId = client.scheduleNewWorkflow(workflowName);
// client.terminate(instanceToTerminateId);
```